rocAL_pybind/examples/rocAL_api_audio_unittest.py

```diff
@@ -138,7 +148,7 @@ def main():
                         print("Audio", audio_tensor)
                         print("Roi", roi)
                     if args.display:
-                        plot_audio_wav(audio_tensor, cnt)
+                        plot_audio_wav(audio_tensor, label)
```

Comment:
Label may not be unique to each file right, That is why we used cnt, Please revert